packages/lib-utils/src/hooks/useWorkspace.ts

```diff
@@ -0,0 +1,51 @@
+import { consoleLogger } from '@openshift/dynamic-plugin-sdk';
+import { useState } from 'react';
+import { getActiveWorkspace, setActiveWorkspace } from '../k8s/k8s-utils';
+
+/**
+ * Hook that retrieves the active workspace from localStorage. The key for the active workspace is
+ * always `sdk/active-workspace`
```

Comment:
For whom it may concern - `sdk/` is consistent with the localstorage prefix we use in API discovery as well. We may want to eventually consolidate the prefix to one const that all location can pull from in case we decide to change it in the future.